### PR TITLE
create uefi img for dd to usb stick

### PR DESCRIPTION
**Pull request by delandtj**
Only thing : not tested, as I don't have access to bootstrap.gig.tech.

so unknown : losetup works on that box ?

### CODE REVIEW

File: scripts/mkuefi.sh [hunk 1]
```diff
@@ -16,4 +16,19 @@ MKTRUST="letsencrypt-x3-cross.crt,letsencrypt-x3.crt"
 
 make bin-x86_64-efi/ipxe.efi EMBED=${root}/boot.ipxe CERT=${MKCERT} TRUST=${MKTRUST}
 
-cp bin-x86_64-efi/ipxe.efi ${root}/
+# create a loop device to hold partitions and UEFI data
+dd if=/dev/zero of=${root}/uefimg.img bs=512 count=4k # large ee-nuff ?
```
Comment (maxux) [inline scripts/mkuefi.sh]: I think `bs=2M count=1` is more readable and will not make any performance difference

File: scripts/mkuefi.sh [hunk 2]
```diff
@@ -16,4 +16,19 @@ MKTRUST="letsencrypt-x3-cross.crt,letsencrypt-x3.crt"
...
+
+parted ${root}/uefimg.img -s "mklabel msdos mkpart primary fat16 0 100%"
+LOOP=`losetup --partscan --find --show ${root}/uefimg.img`
```
Comment (maxux) [inline scripts/mkuefi.sh]: Please use `$(...)` for executing commands

File: scripts/mkuefi.sh [hunk 3]
```diff
@@ -16,4 +16,19 @@ MKTRUST="letsencrypt-x3-cross.crt,letsencrypt-x3.crt"
 
 make bin-x86_64-efi/ipxe.efi EMBED=${root}/boot.ipxe CERT=${MKCERT} TRUST=${MKTRUST}
 
-cp bin-x86_64-efi/ipxe.efi ${root}/
+# create a loop device to hold partitions and UEFI data
+dd if=/dev/zero of=${root}/uefimg.img bs=512 count=4k # large ee-nuff ?
+
+parted ${root}/uefimg.img -s "mklabel msdos mkpart primary fat16 0 100%"
```
Comment (maxux) [inline scripts/mkuefi.sh]: Does parted need extra stuff to make fat16 partition or is it built-in ?

Comment (delandtj) [inline scripts/mkuefi.sh]: built-in

File: scripts/mkuefi.sh (hunk 1 above)
Comment (delandtj) [inline scripts/mkuefi.sh]: aye, your call

File: scripts/mkuefi.sh (hunk 2 above)
Comment (delandtj) [inline scripts/mkuefi.sh]: ait.. why ?

**Pull request merged**